### PR TITLE
Toggle layer with no visible children

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/LayerTree/LayerTree.js [hunk 1]
```diff
@@ -93,9 +93,19 @@ const defaultProps = {
 class LayerTree extends Component {
   constructor(props) {
     super(props);
+    const { layerService, isItemHidden } = this.props;
+    const initialExpandedLayerNames =
```
Comment (mhaertwig) [inline src/components/LayerTree/LayerTree.js]: Couldn't we use the layer instances itself instead of their keys (`expandedLayers`)?

Comment (edellucien) [reply src/components/LayerTree/LayerTree.js]: Both are working. I do it with the layer instance instead.

File: src/components/LayerTree/LayerTree.js [hunk 2]
```diff
@@ -93,9 +93,18 @@ const defaultProps = {
...
+      layerService && layerService.getLayers()
+        ? layerService
+            .getLayers()
+            .filter(l => !isItemHidden(l))
+            .filter(l => l.getVisibleChildren().length)
```
Comment (oterral) [inline src/components/LayerTree/LayerTree.js]: one filter to rules them all
```
.filter(l => !isItemHidden(l) &&l.getVisibleChildren().length )
```

Comment (edellucien) [reply src/components/LayerTree/LayerTree.js]: better so!

**Comment by edellucien** [comment]
Ready to be merged

**Pull request merged**